community-membership.md

```diff
@@ -3,21 +3,20 @@
 **Note:** This document is in progress
 
 This doc outlines the various responsibilities of contributor roles in Kubernetes.  The Kubernetes
-project is subdivided into various sub-areas.  Responsibilities for many contributor roles are
-scoped by these sub-areas.
+project is subdivided into various sub-areas.  Responsibilities for most roles is scoped to these sub-areas.
 
 | Role | Responsibilities | Requirements | Defined by |
 | -----| ---------------- | ------------ | -------|
 | member | active contributor in the community | sponsored by 2 reviewers.  multiple contributions to the project. | Kubernetes GitHub org member. |
-| reviewer | review contribution from other members | history of review and authorship in an area | OWNERS file reviewer entry. |
-| approver | approve accepting contributions | highly experienced and active reviewer + contributor to an area | OWNERS file approver entry|
-| owner | set priorities and approve proposals | demonstrated responsibility and good judgement for entire area | OWNERS file approver entries for entire area.  "# owner" comment next to entry. |
-| maintainer | cross area ownership of project health | highly experienced contributor active in multiple areas and roles. | GitHub repo write access |
+| reviewer | review contributions from other members | history of review and authorship in an area | [OWNERS] file reviewer entry. |
+| approver | approve accepting contributions | highly experienced and active reviewer + contributor to an area | [OWNERS] file approver entry|
+| owner | set direction and priorities for a subarea of the project | demonstrated responsibility and excellent technical judgement for the area | [sigs.yaml] subproject [OWNERS] file *owners* entry |
 
 ## New contributors
 
-[**New contributors**](https://github.com/kubernetes/community/blob/master/CONTRIBUTING.md) should be welcomed to the community
-by existing members, helped with PR workflow, and directed to relevant documentation and communication channels.
+[New contributors] should be welcomed to the
+community by existing members, helped with PR workflow, and directed to relevant documentation and communication
```

Comment:
Since the adjacent lines were modified anyway, I suggest reformatting this paragraph.